### DISCUSSION TITLE
2022-05-18 Weekly Meeting 56 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#235] [Weekly Meetings]
Weekly Meeting 56 is happening on Wednesday May 18th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 
- New meeting format: demos from the community?

Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
  - blog update
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - First developer hour! 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
- [Media Group](https://github.com/omigroup/media)
- any others?

Demos? / Topics?:

**Comment by robertlong** [comment]
# Notes

- Wednesday meeting format:
  - Switch to more events, less structure, more fun, but regular cadence
- Developer Hour:
  -  Talked about repackaging Spoke editor components for use in other React 3D editor tooling
  - Lyuma showed off Unity project import into Godot and export from Godot to glTF!
  - Jin showed off Cryptovoxels import / export into Blender and other platforms
  - Happening every other week 9:00 AM PST
- omi-media-howto channel
  - New channel for how to get started in the media group
  - Looking for video editors, blog contributors, etc.
- [glTF Working Group](https://github.com/omigroup/gltf-extensions/discussions/83)

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/media/issues/38

**Comment by mrmetaverse** [comment]
Media group to create intake form for submitting blogs and blurbs 

Once first blog is published, we can start pushing out mini pieces like "how the scripting group discovered the value of open dev chats", "upcoming events" etc.
